test/controllers/organizations/staff/activations_controller_test.rb

```diff
@@ -0,0 +1,34 @@
+require "test_helper"
+require "action_policy/test_helper"
+
+class Organizations::ActivationsControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @organization = ActsAsTenant.current_tenant
+    @staff = create(:admin)
+    sign_in @staff
+  end
+
+  context "authorization" do
+    include ActionPolicy::TestHelper
+
+    context "#update_activation" do
```

Comment:
do we need this authorization test here? I think we should leave authorization tests to the dedicated authorization test files, and you have an activations_policy_test file